### PR TITLE
task02 and task01(already in branch)

### PR DESCRIPTION
**Pull request by AntsiferauGodel**
task02 and task01(already in branch)

### CODE REVIEW

File: Unit-04/SQLQueryTask2.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
+SELECT TOP 8 DepartmentID,Name,GroupName,ModifiedDate FROM AdventureWorks2017.HumanResources.Department ORDER BY Name DESC;
```
Comment (HannaKeryna) [inline Unit-04/SQLQueryTask2.sql]: You can add "USE" on the top of the script

File: Unit-04/SQLQueryTask2.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+SELECT NationalIDNumber,BusinessEntityID,JobTitle,BirthDate,HireDate FROM AdventureWorks2017.HumanResources.Employee WHERE DATEDIFF(YEAR, BirthDate,HireDate) = 22;
```
Comment (HannaKeryna) [inline Unit-04/SQLQueryTask2.sql]: Please format script like
SELECT ...
FROM ..
WHERE ..

Please update in all file

Comment (AntsiferauGodel) [reply Unit-04/SQLQueryTask2.sql]: done

File: Unit-04/SQLQueryTask1.sql [hunk 3]
```diff
@@ -1,8 +1,9 @@
 CREATE DATABASE Something;
+GO
 SELECT name,create_date FROM master.sys.databases;
 CREATE TABLE Something.dbo.Wicked (
 Id int NOT NULL,
 );
 BACKUP DATABASE Something TO DISK = 'D:\GitLabMastery\GodelMastery\Unit-04\Something.bak';
```
[the remark this answers is not in the document]
Comment (AntsiferauGodel) [reply Unit-04/SQLQueryTask1.sql]: done

File: Unit-04/SQLQueryTask2.sql (hunk 1 above)
Comment (AntsiferauGodel) [reply Unit-04/SQLQueryTask2.sql]: done

File: Unit-04/SQLQueryTask2.sql [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+FROM HumanResources.Employee
+WHERE MaritalStatus = 'M' AND JobTitle IN ('Design Engineer' , 'Tool Designer', 'Engineering Manager', 'Production Control Manager') 
+ORDER BY BirthDate ASC;
+--2.4
+SELECT BusinessEntityID,JobTitle,Gender,BirthDate,HireDate 
+FROM HumanResources.Employee
+WHERE (MONTH(HireDate) = 3 AND DAY(HireDate) = 5) 
+ORDER BY BusinessEntityID ASC OFFSET 1 ROWS FETCH NEXT 5 ROWS ONLY;
+--2.5
+SELECT  BusinessEntityID,JobTitle,Gender,HireDate,REPLACE(LoginID,'adventure-works','adventure-works2017')
```
Comment (HannaKeryna) [inline Unit-04/SQLQueryTask2.sql]: Please add name of the column

**Pull request merged**